Items/Armor/DesertProwler/DesertProwlerSet.cs

```diff
@@ -166,7 +170,8 @@ public static void ModifySetTooltips(ModItem item, List<TooltipLine> tooltips)
 
                 if (setBonusIndex != -1)
                 {
-                    TooltipLine setBonus1 = new TooltipLine(item.Mod, "CalamityMod:SetBonus1", CalamityUtils.GetTextValueFromModItem<DesertProwlerHat>("AbilityBrief"));
+                    string dir = Main.ReversedUpDownArmorSetBonuses ? "UP" : "DOWN";
```

Comment:
Does vanilla have localization for UP/DOWN?

Reply:
Yes, this is identical to how it's handled in vanilla. "UP" with the setting enabled, "DOWN" without.

Reply:
Are these identical or not? I asked if vanilla has localization and was met with "yes", then followed by a declaration that it is identical despite not being localized.